src/lib/settings/SettingsFolder.js

```diff
@@ -268,6 +269,7 @@ class SettingsFolder extends Map {
 					const keys = options.onlyConfigurable ? entry.configurableKeys : [...entry.keys()];
 					throw keys.length ? language.get('SETTING_GATEWAY_CHOOSE_KEY', keys.join('\', \'')) : language.get('SETTING_GATEWAY_UNCONFIGURABLE_FOLDER');
 				}
+				// TODO(kyranet): Time to remember what this was
```

Comment:
If the key was not an array, but the value passed was an array (particularly in prefix where it can change)